packages/framework/presence/src/systemWorkspace.ts

```diff
@@ -7,11 +7,12 @@ import { assert } from "@fluidframework/core-utils/internal";
 
 import type { ClientConnectionId } from "./baseTypes.js";
 import type { InternalTypes } from "./exposedInternalTypes.js";
-import type {
-	ClientSessionId,
-	IPresence,
-	ISessionClient,
-	PresenceEvents,
+import {
+	SessionClientStatus,
+	type ClientSessionId,
+	type IPresence,
+	type ISessionClient,
+	type PresenceEvents,
 } from "./presence.js";
```

Comment:
Keep type imports split from non-type imports. That is the preferred pattern and what is used throughout presence package.